proxies/live/apiproxy/resources/jsc/RestrictRequest.js

```diff
@@ -3,12 +3,14 @@ var request_pathsuffix = context.getVariable('proxy.pathsuffix');
 var request_path_nhs_number = request_pathsuffix.split("/")[2];
 var method_is_restricted = true
 var sync_wrapped = context.getVariable('request.header.x-sync-wrapped');
+var vot = context.getVariable('jwt.DecodeJWT.DecodeIdToken.claim.vot');
+var allowed_vots = ["P9.Cp.Cd","P9.Cm","P9.Cp.Ck"];
 
 if (request_pathsuffix.split("/")[1] == "_poll" && sync_wrapped == "true"){
     method_is_restricted = false
 }
 
-if (request_path_nhs_number == id_token_nhs_number){
+if ((request_path_nhs_number == id_token_nhs_number) && (allowed_vots.indexOf(vot) > -1)) {
```

Comment:
```suggestion
if ((request_path_nhs_number == id_token_nhs_number) && allowed_vots.includes(vot)) {
```
Might only make a functional difference when dealing with 0, undefined, NaN (so won't matter for our usage), mostly for readbility, for your consideration

Reply:
I did start off with 'includes' but found it did not work very well and failed certain karate tests, so although includes more readable, I'd probably be inclined to stick with indexOf